web/src/components/inventory/InventoryContext.tsx

```diff
@@ -65,7 +65,7 @@ const InventoryContext: React.FC<{
           <Item onClick={handleClick} data={{ action: 'drop' }}>
             {Locale.ui_drop}
           </Item>
-          {props.item.name.startsWith('WEAPON_') && props.item.metadata?.serial && (
+          {props.item.metadata?.serial && (
```

Comment:
How'd this work? Not all weapons have a serial 🤔

Reply:
Fixed, I added new change where it checks with hashname. @LukeWasTakenn